index.html

```diff
@@ -5,41 +5,242 @@
-    <title>portfolio</title>
+    <title>Portfolio</title>
+    <!-- font awesome -->
     <script src="https://kit.fontawesome.com/6b55e5dec8.js" crossorigin="anonymous"></script>
+    <!-- css link -->
     <link rel="stylesheet" href="styles.css">
 </head>
 
 <body>
-    <!-- this is the navigation header -->
+
+
+    <!-- Navigation header -->
     <header>
         <div class="navigation">
             <a href="">WELCOME</a>
             <nav class="toggle">
-           <button class="menu"><i class="fa-solid fa-bars"></i></button>
+                <button class="menu"><i class="fa-solid fa-bars"></i></button>
             </nav>
         </div>
     </header>
 
     <!-- this is the section part -->
 
-    <section class="first-section">
-
-        <div class="section2">
-            <h1>I'm Ekene, <br> glad to see <br> you!</h1>
-            <p class="first-paragraph">I'm a software developer! I can help you build a product , feature or website Look through 
-                some of my work and experience! If you like what you see and have a project you need coded,
-                don't hesitate to contact me.</p>
-            <p class="second-paragraph">LETS CONNECT</p>
-            <ul class="social-media">
-                <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
-                <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
-                <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
-                <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
-            </ul>
-        </div>
-    </section>
+    <div class="background-color-container">
+
+
+        <section class="first-section">
+
+            <div class="section2">
+                <h1>I'm Ekene, <br> glad to see <br> you!</h1>
+                <p class="first-paragraph">I'm a software developer! I can help you build a product , feature or website
+                    Look through
+                    some of my work and experience! If you like what you see and have a project you need coded,
+                    don't hesitate to contact me.</p>
+                <p class="second-paragraph">LETS CONNECT</p>
+                <ul class="social-media">
+                    <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
+                    <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
+                    <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
+                    <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
+                </ul>
+            </div>
+        </section>
+
+        <div class="grid-position">
+            <div class="Grid-container">
+                <section class="second-section">
+                    <img src="./images/Snapshoot-Portfolio-(3).jpg" alt="section-eader">
+                    <h1>TONIC</h1>
```

Comment:
Kindly change the heading of the project to be any heading from `h2` to `h6`, for example: `h2` instead of `h1`. since the `h1` tag should only exist once on your page. Please do this for all projects